Lib/fontTools/varLib/avar.py

```diff
@@ -0,0 +1,72 @@
+from fontTools.varLib import _add_avar, load_designspace
+from fontTools.misc.cliTools import makeOutputFileName
+import logging
+
+log = logging.getLogger("fontTools.varLib.avar")
+
+
+def main(args=None):
+    """Add `avar` table from designspace file to variable font."""
+
+    if args is None:
+        import sys
+
+        args = sys.argv[1:]
+
+    from fontTools import configLogger
+    from fontTools.ttLib import TTFont
+    from fontTools.designspaceLib import DesignSpaceDocument
+    import argparse
+
+    parser = argparse.ArgumentParser(
+        "fonttools varLib.avar",
+        description="Add `avar` table from designspace file to variable font.",
+    )
+    parser.add_argument("font", metavar="varfont.ttf", help="Variable-font file.")
+    parser.add_argument(
+        "designspace", metavar="family.designspace", help="Designspace file."
+    )
+    parser.add_argument(
+        "-o",
+        "--output-file",
+        type=str,
+        help="Output font file name.",
+    )
+
+    logging_group = parser.add_mutually_exclusive_group(required=False)
```

Comment:
mutually exclusive group but contains only one option as far as I can see..

Reply:
Humm, yeah. There wasn't a point to add `--quiet`. So I just remove the `logging_group` completely?

Reply:
ok

Reply:
Fixed.